rules/make-newcomers-feel-welcome/rule.md

```diff
@@ -59,7 +59,7 @@ Other nice ideas could include:
 * Use a blank notebook to write a welcome note and put handy hints inside, then they can continue to use the notepad during induction! 📓
 
 ::: good
-![Figure: Good example - A personal, fun welcome to SSW Sydney with all of the good food and coffee tips!](Penny Welcome Pic.jpg)
+![Figure: Good example - A fun welcome to SSW Sydney with all of the good food and coffee places in an AOI map!](MicrosoftTeams-image (70).png)
```

Comment:
Hey @LuciaSirtori 

1. Could you please rename (and reupload) this image? It should have a descriptive name

- Avoid spaces
- Avoid the generic name (MicrosoftTeams-image...) We had problem with this before

As per https://www.ssw.com.au/rules/how-to-name-documents